modules/core/src/webgl/classes/resource.js

```diff
@@ -284,4 +289,14 @@ export default class Resource {
 
     stats.resourceMap[name].active--;
   }
+
+  _trackAllocatedMemory(bytes) {
```

Comment:
Nit: Probably a lot of extra work, but assume it could be valuable to see what type of resource is using the bytes? I.e. add one more param?

Reply:
That's what's happening in the overrides in the subclasses.